docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc

```diff
@@ -0,0 +1,224 @@
+// tag::cloud[]
+In order to get the shards assigned we'll need to call the 
+<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
+resolve the conflicting routing configurations towards using the standardised
```

Comment:
```suggestion
In order to get the shards assigned we need to call the 
<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
resolve the conflicting routing configurations towards using the standardized
```

(don't blame me, blame the Elastic style guide for proscribing American spellings for things)